android/src/net/hockeyapp/android/NativeCrashManager.java

```diff
@@ -1,100 +1,180 @@
 package net.hockeyapp.android;
```

Comment:
Should this code not be somewhere under com.eegeo.mobileexampleapp (or at least changed to com.microsoft.appcenter) - although I think as it's *our* code it should live in our package - unless it needs to be here for some reason I don't understand.

Reply:
Yes. it should be in our package. Previously we were using the same package of hockey app and added this native crash manager there. As that package has been removed it can be a part of our app now.